chaos-monkey-spring-boot/pom.xml

```diff
@@ -31,6 +31,11 @@
     </parent>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-context</artifactId>
+            <scope>test</scope>
```

Comment:
Version is missing. Eg.  
```xml
<version>2.1.3.RELEASE</version>
```

Reply:
I am not sure that we need the actual version here. If you look at all other dependencies - junit for example?